docroot/modules/humsci/hs_layouts/hs_layouts.module

```diff
@@ -7,15 +7,29 @@
 
 use Drupal\Component\Utility\Html;
 use Drupal\paragraphs\ParagraphInterface;
+use Drupal\dashboard\Plugin\SectionStorage\DashboardSectionStorage;
 
 /**
  * Implements hook_plugin_filter_TYPE__CONSUMER_alter().
  */
 function hs_layouts_plugin_filter_layout__layout_builder_alter(&$definitions, &$extra, &$consumer) {
   /** @var \Drupal\Core\Layout\LayoutDefinition $definition */
   foreach ($definitions as $id => $definition) {
-    if ($definition->getProvider() !== 'hs_layouts') {
-      unset($definitions[$id]);
+    $provider = $definition->getProvider();
+    if ($extra['section_storage'] instanceof DashboardSectionStorage) {
+      // Define providers to unset for the gin admin theme.
+      $admin_providers_to_unset = [
```

Comment:
Try to keep variable names under 20 char, otherwise we'll get a CodeClimate error.